docs/manual/output.md

```diff
@@ -76,6 +76,26 @@ output schemes are registered, but no file will be created.
 
 :::
 
+In case a multithreaded simulation is requested with the `-t` or `--threads`
+option (see {ref}`running`), the output file names with be appended with the
+thread number. _remage_ will produce one output file per thread appending
+`_t$id` where `$id` is the thread number before the file extension.
+
+For example running _remage_ with:
+
+```
+remage -o OUTPUT.lh5 -t 8
+```
+
+will result in output files `OUTPUT_t0.lh5,..., OUTPUT_t7.lh5`.
+
+For the LH5 output format it is possible to merge the output files before saving
+to disk, this can be enabled with the `--merge-output-files` (or `-m`) option.
```

Comment:
here you have to mention that G4 automatically merges files for all (check) other formats. Only for HDF5 merging is not supported.